.github/workflows/docker-build.yml

```diff
@@ -20,5 +20,5 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Build image
-        run: docker build . --file Dockerfile
+        run: docker build -t testcontainers/helloworld:latest . --file Dockerfile
```

Comment:
I assume this does not really matter, since it will be deployed through Docker Hub autobuild.

Reply:
I added it as a demonstration on how to build the project, which is not documented. Probably worth it documenting it?

Reply:
Sure. Also, we learned Autobuild does not work anymore in Free Team plan of Docker Hub, so we'll likely need to replicate the other pipelines after all.